pages/common/figlet.md

```diff
@@ -0,0 +1,15 @@
+# figlet
+
+> Display large characters made up of ordinary screen characters.
+
+- Use default settings:
+
+`figlet {{Hello World}}`
+
+- To change the font, use the -f option:
```

Comment:
The usage of the `-f` option is pretty clear from the command itself. So no need to mention it again in the description.

How about - `Generate output by specifying a given font`